app/test_main.py

```diff
@@ -1,3 +1,68 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        pytest.param(
+            0, 0, [0, 0],
+            id="Should return zero if cat or dog age equal to zero"
+        ),
+        pytest.param(
+            14, 14, [0, 0],
+            id="Should return zero if cat or dog age less than 15"
```

Comment:
This test case description is misleading. The test checks if the function returns zero when the age is less than 15, not just 14. Consider updating the description to reflect this.